wordpress-mobile/WordPress-Android
# Hide author selection filter for users who can't see posts of others

Fixes #9548 

Note: #9581 needs to be merged first + update target branch to "feature/master-post-filters".

To test:
1. Switch to a site where you are in Contributor role
2. My Site
3. Blog Posts
4. Notice the Author selection filter is hidden

Repeat 1-4 for Author role
Repeat 1-3 for Editor and Administrator roles and notice the author selection filter is visible

Update release notes:

We'll update the release notes when we merge "feature/master-post-filters" into develop.
